frontend/public/components/image-stream.tsx

```diff
@@ -107,6 +109,31 @@ const ImageStreamTagsRow: React.SFC<ImageStreamTagsRowProps> = ({imageStream, sp
   </div>;
 };
 
+const ImageStreamManipulationHelp: React.SFC<ImageStreamManipulationHelpProps> = ({registryAddress, isRegistryPublic}) => {
+  const loginCommand = isRegistryPublic ? 'oc registry login' : `podman login ${_.split(registryAddress, '/', 1)[0]} -u $(oc whoami) -p $(oc whoami -t)`;
```

Comment:
We should only show the popover when the image registry is public. The podman command wouldn't work either if not public.